recipes/libxml2/all/conanfile.py

```diff
@@ -215,113 +250,101 @@ def _build_mingw(self):
             if not self.options.shared:
-                os.remove(os.path.join(self.package_folder, "bin", "libxml2.dll"))
-            os.remove(os.path.join(self.package_folder, "lib", "libxml2_a_dll.lib"))
-            os.remove(os.path.join(self.package_folder, "lib", "libxml2_a.lib" if self.options.shared else "libxml2.lib"))
-            tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "*.pdb")
+                rm(self, "libxml2.dll", os.path.join(self.package_folder, "bin"))
+            rm(self, "libxml2_a_dll.lib", os.path.join(self.package_folder, "lib"))
+            rm(self, "libxml2_a.lib" if self.options.shared else "libxml2.lib", os.path.join(self.package_folder, "lib"))
+            rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
         elif self._is_mingw_windows:
             self._package_mingw()
             if self.options.shared:
-                os.remove(os.path.join(self.package_folder, "lib", "libxml2.a"))
-                tools.rename(os.path.join(self.package_folder, "lib", "libxml2.lib"),
+                rm(self, "libxml2.a", os.path.join(self.package_folder, "lib"))
+                rename(self, os.path.join(self.package_folder, "lib", "libxml2.lib"),
                              os.path.join(self.package_folder, "lib", "libxml2.dll.a"))
             else:
-                os.remove(os.path.join(self.package_folder, "bin", "libxml2.dll"))
-                os.remove(os.path.join(self.package_folder, "lib", "libxml2.lib"))
+                rm(self, "libxml2.dll", os.path.join(self.package_folder, "bin"))
+                rm(self, "libxml2.lib", os.path.join(self.package_folder, "lib"))
         else:
-            autotools = self._configure_autotools()
-            autotools.make(["install-libLTLIBRARIES", "install-data"])
+            autotools = Autotools(self)
+
+            for target in ["install-libLTLIBRARIES", "install-data"]:
+                autotools.make(target=target, args=[f"DESTDIR={unix_path(self, self.package_folder)}"])
 
             if self.options.include_utils:
-                autotools.make(["install", "xmllint", "xmlcatalog", "xml2-config"])
+                autotools.install()
 
-            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
-            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.sh")
+            rm(self, "*.la", os.path.join(self.package_folder, "lib"))
+            rm(self, "*.sh", os.path.join(self.package_folder, "lib"))
             for prefix in ["run", "test"]:
-                tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), prefix + "*")
-            tools.rmdir(os.path.join(self.package_folder, "share"))
-            tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
-            tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+                rm(self, prefix + "*", os.path.join(self.package_folder, "bin"))
```

Comment:
I mean for two lines just copy past instead of a for loop? (just a super minor nit)

Reply:
Indeed, but I didn't adjust that as I'm looking to make minimum-change patches where possible...

Reply:
I fixed this.